src/PhraseApp.php

```diff
@@ -164,12 +166,13 @@ public function update(Message $message)
     /**
      * {@inheritdoc}
      */
-    public function delete($locale, $domain, $key)
+    public function delete(string $locale, string $domain, string $key): void
     {
         /* @var KeySearchResults $results */
-        $results = $this->client->key()->search($this->projectId, $this->getLocaleId($locale), [
+        $results = $this->client->key()->search($this->projectId, [
             'tags' => $domain,
-            'name' => $domain.'::'.$key
+            'name' => $domain.'::'.$key,
+            'ids' => $this->getLocaleId($locale),
```

Comment:
`ids` seems to mean array of ids are expected? Is it ok to pass a string?

Reply:
I replaced `ids` with `locale_id` after checking the API doc (https://developers.phrase.com/api/#post-/projects/{project_id}/keys/search)